esp/scm/ws_espcontrol.ecm

```diff
@@ -17,13 +17,23 @@
 
 EspInclude(common);
 
+ESPenum LogLevelEnum : int
+{
+    None(0, "none"),
+    Min(1, "min"),
+    Normal(5, "normal"),
+    Max(10, "max"),
+};
+
 ESPrequest [nil_remove] SetLoggingRequest
 {
-    int LoggingLevel;
+    [max_ver("1.03")] int LoggingLevel;
```

Comment:
Change to [depr_ver("1.04")] int LoggingLevel for accuracy

Reply:
Changed.